auth-api/email_templates/govm_business_invitation_email.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
```

Comment:
template is just put in so that the code wont break..we wont have the template ready for now